src/sagas/wallet.js

```diff
@@ -447,6 +448,25 @@ export function* listenForWalletReady(wallet) {
   }
 }
 
+/**
+ * An abstraction effect to call wallet's checkAddressMine method.
+ * It subjects the call to a retry strategy and return the result of the request,
+ * which can be either success or error.
+ *
+ * @param {Object} wallet Either fullnode or wallet-service wallet.
+ * @param {string[]} txAddresses A collection of addresses to check.
+ * @returns {{
+ *   success?: { [address: string]: boolean };
+ *   error?: any;
+ * }} If success, it returns a map of type address:isMine,
+ * otherwise it will return the error definition.
+ */
+function* checkAddressMine(wallet, txAddresses) {
```

Comment:
Why don't you just pass wallet.checkAddressMine to the `progressiveRetryRequest` method it in the `handleTx` method instead of implementing this abstraction with the same name?

Reply:
Removed.